docs/docs/location-data-from-props.md

```diff
@@ -24,6 +24,35 @@ Note that you have to parse the `search` field (the [query string](https://devel
 
 Using `hash` in JavaScript is one way to update the browser URL and the DOM without having the browser do a full HTML page reload. HashHistory in `@reach/router` is used to track browser history with JavaScript when using [hashrouter](https://reacttraining.com/react-router/web/api/HashRouter) instead of [browserrouter](https://reacttraining.com/react-router/web/api/BrowserRouter) which uses the newer HTML5 `history` API.
 
+### Getting the absolute URL of a page
+
+The location properties generally do not include the domain of your site, since Gatsby does know where you will deploy it.
```

Comment:
```suggestion
The `location` object's properties generally do not include the domain of your site, since Gatsby doesn't know where you will deploy it.
```